cli/README.md

```diff
@@ -643,6 +644,10 @@ This is the same parameter as the one described in the [above section](#disabled
 
 Specifies whether you want to run `cordova build` instead of `cordova prepare` (which is the default behavior), when generating your updated web assets. This is valuable if your project includes before and/or after build hooks (e.g. to transpile TypeScript), and therefore, having CodePush simply run `cordova prepare` isn't sufficient to create and release an update. If left unspecified, this defaults to `false`.
 
+#### IsReleaseBuildType parameter
```

Comment:
Both the `Build` and `IsReleaseBuildType` sections need to be alphabetically ordered above